lib/shopify_api/only_send_mutated_params.rb

```diff
@@ -17,6 +10,13 @@ def method_missing(method_symbol, *args)
       super
     end
 
+    def save
+      slice_attributes_on_changed
+      base = super
+      clear_changed_attributes
```

Comment:
Ill write a test to prove it but afaik `super` here will throw an exception and the flow will stop